README.md

```diff
@@ -22,13 +26,67 @@ When the reader has completed this Code Pattern, they will understand how to:
 * Generate captions for an image using the MAX Model's REST API
 * Run a web application that using the model's REST API
 
+<!-- TODO: Insert Arch diagram -->
+
+## Flow
+
+<!-- TODO: Add Flow Steps -->
+
+## Included Components
+
+* [IBM Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/): A place for developers to find and use
+free and open source deep learning models.
+
+## Featured Technologies
+
+* [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate
+your systems more effectively.
+
+* [Bootstrap 3](https://getbootstrap.com): Bootstrap is a free and open-source front-end library for designing websites
+and web applications.
+
+* [D3.js](https://d3js.org): D3.js is a JavaScript library for manipulating documents based on data.
+
+* [d3-cloud](https://github.com/jasondavies/d3-cloud): A Wordle-inspired word cloud layout written in JavaScript.
+
+* [Featherlight](https://noelboss.github.io/featherlight/): Featherlight is a very lightweight jQuery lightbox plugin.
+
+* [Glyphicons](http://glyphicons.com): GLYPHICONS is a library of precisely prepared monochromatic icons and symbols,
+created with an emphasis to simplicity and easy orientation.
+
+* [Image Picker](http://rvera.github.io/image-picker/): Image Picker is a simple jQuery plugin that transforms a select
+element into a more user friendly graphical interface.
+
+* [JQuery](https://jquery.com): jQuery is a cross-platform JavaScript library designed to simplify the client-side
+scripting of HTML.
+
+* [Pexels](https://www.pexels.com): Pexels provides high quality and completely free stock photos licensed under the
```

Comment:
I think we want to move this to the license section? We just want to say the license for the images are CC0.

Reply:
Remove spaces between each bullet.

Reply:
According to their website we don't even need to include this line (for licensing purposes). I'm only including it here as a reference to it's usage. It may be clearer to remove reference to it's licensing in the description here.

Reply:
I can remove the spaces, I added them to match the example readme's

Reply:
Also do we need to list all the libraries we used? Or just focus on the main components like Docker?

Reply:
We don't *have to* list them all, but it's better to. And I should actually add Docker as a component now that you mention it